api/Dockerfile

```diff
@@ -1,24 +1,30 @@
+# Remember: the context for this build is the root of the monorepo,
+# not the api folder
+
+# TODO: we only use bun as a dependency manager, but api is still executed with
+# node. We should eventually get rid of node
 FROM oven/bun:slim as base
 WORKDIR /usr/src/app
 
 FROM node:slim as runtime
 WORKDIR /usr/src/app
 RUN apt-get update -y && apt-get install -y openssl
 
-# install dependencies into temp directory
-# this will cache them and speed up future builds
+# Install dependencies into temp directory
+# This will cache them and speed up future builds
 FROM base AS install
-# install openssl for prisma
-RUN apt-get update -y && apt-get install -y openssl 
-# install
-RUN mkdir -p /temp/dev
-COPY package.json bun.lockb prisma/ /temp/dev/
+# Install openssl for prisma
+RUN apt-get update -y && apt-get install -y openssl
+RUN mkdir -p /temp/dev/api
+COPY package.json bun.lockb /temp/dev/
+COPY api/package.json /temp/dev/api/package.json
```

Comment:
`api/package.json` specifies all the prod dependencies actually used by `api`; `frontend/package.json` is needed just because the root package references it. I think this is the minimal number of files needed to install.